authorization/src/main/java/com/terabite/GlobalConfiguration.java

```diff
@@ -1,25 +1,30 @@
 package com.terabite;
 
+import org.springframework.beans.factory.annotation.Qualifier;
 import org.springframework.beans.factory.annotation.Value;
 import org.springframework.context.annotation.Bean;
 import org.springframework.context.annotation.Configuration;
 
 import com.terabite.authorization.AuthorizationApi;
+import com.terabite.authorization.repository.LoginRepository;
+import com.terabite.authorization.service.JwtService;
 import com.terabite.user.UserApi;
 
 @Configuration
 public class GlobalConfiguration {
 
 	public static final String BEAN_NAME_AUTH_COOKIE_NAME = "authCookieName";
 	public static final String BEAN_NAME_DOMAIN_URL = "domainUrl";
```

Comment:
Just making a note here. Cookies get created w/ domain localhost, which may break existing Insomnia tests. Need to change 127.0.0.1 to localhost in future Insomnia tests

Reply:
noted!